docs/source/tracking/tracking-api.rst

```diff
@@ -0,0 +1,281 @@
+By default, ``timestamp`` is set to the current time and ``step`` is set to 0. You can override these values by passing
+``timestamp`` and ``step`` arguments to the ``log`` methods. For example:
+
+.. code-block:: python
+
+    mlflow.log_metric(key="train_loss", value=train_loss, step=epoch, timestamp=now)
+
+``step``  has the following requirements and properties:
+
+- Must be a valid 64-bit integer value.
+- Can be negative.
+- Can be out of order in successive write calls. For example, (1, 3, 2) is a valid sequence.
+- Can have "gaps" in the sequence of values specified in successive write calls. For example, (1, 5, 75, -20) is a valid sequence.
+
+If you specify both a timestamp and a step, metrics are recorded against both axes independently.
+
+.. _organizing_runs_in_experiments:
+
+Organizing Runs in Experiments
+------------------------------
+
+MLflow allows you to group runs under experiments, which can be useful for comparing runs intended
+to tackle a particular task. You can create experiments via multiple way - MLflow UI, the :ref:`cli` (``mlflow experiments``),
+or the :py:func:`mlflow.create_experiment` Python API. You can pass the experiment name for an individual run
+using the CLI (for example, ``mlflow run ... --experiment-name [name]``) or the ``MLFLOW_EXPERIMENT_NAME``
+environment variable. Alternatively, you can use the experiment ID instead, via the
+``--experiment-id`` CLI flag or the ``MLFLOW_EXPERIMENT_ID`` environment variable.
+
+.. code-block:: bash
+
+    # Set the experiment via environment variables
+    export MLFLOW_EXPERIMENT_NAME=fraud-detection
+
+    mlflow experiments create --experiment-name fraud-detection
+
+.. code-block:: python
+
+    # Launch a run. The experiment is inferred from the MLFLOW_EXPERIMENT_NAME environment
+    # variable, or from the --experiment-name parameter passed to the MLflow CLI (the latter
+    # taking precedence)
+    with mlflow.start_run():
+        mlflow.log_param("a", 1)
+        mlflow.log_metric("b", 2)
+
+.. _launching-multiple-runs:
+
+Launching Multiple Runs in One Program
+--------------------------------------
+Sometimes you want to launch multiple MLflow runs in the same program: for example, maybe you are
+performing a hyperparameter search locally or your experiments are just very fast to run. This is
+easy to do because the ``ActiveRun`` object returned by :py:func:`mlflow.start_run` is a Python
+`context manager <https://docs.python.org/2.5/whatsnew/pep-343.html>`_. You can "scope" each run to
+just one block of code as follows:
+
+.. code-block:: python
+
+   with mlflow.start_run():
+       mlflow.log_param("x", 1)
+       mlflow.log_metric("y", 2)
+       ...
+
+The run remains open throughout the ``with`` statement, and is automatically closed when the
+statement exits, even if it exits due to an exception.
+
+
```

Comment:
Can we show a parent + multiple child runs as an example and explain that it's common to use this setup for hyperparameter tuning?

Reply:
Absolutely! Indeed I feel that might be the main ask of the said question.